resources/monitoring/templates/grafana/kyma-dashboards/kyma-backends.yaml

```diff
@@ -110,9 +113,9 @@ data:
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum by(container) (container_memory_usage_bytes{namespace=\"$namespace\", pod=\"$pod\", container_name!=\"POD\",container!=\"\"})",
+              "expr": "sum by(name, container) (container_memory_working_set_bytes{namespace=\"$namespace\", pod=\"$pod\", container_name!=\"POD\",container!=\"\"})",
               "interval": "",
-              "legendFormat": "Current: {{`{{container_name}}`}}",
+              "legendFormat": "Current: {{`{{name}}`}}",
```

Comment:
Shouldn't we display the container_name indeed. It is much prettier and usefull then this long name. Yes, there will be duplicates but they have different coloring.

Reply:
will fix it